src/dbinterface/DBInterface.cpp

```diff
@@ -0,0 +1,127 @@
+
+#include "DBInterface.h"
+#include "../util/logger/Logger.h"
+
+Logger interface_logger;
+
+int DBInterface::finalize() {
+    return sqlite3_close(database);
+}
+
+typedef vector<vector<pair<string, string>>> table_type;
+
+static int callback(void *ptr, int argc, char **argv, char **azColName) {
```

Comment:
Can we have a better variable name than `azColName`? We need to enhance the meaning of that variable.

Reply:
Ok